src/client/editor/Editor.js

```diff
@@ -886,6 +886,16 @@ export default class Editor {
     const scene = this.scene;
     const clonedScene = scene.clone();
 
+    // Add a preview camera to the exported GLB if there is a transform in the metadata.
+    const { previewCameraTransform } = this.getSceneMetadata();
+
+    if (previewCameraTransform) {
+      const previewCamera = this.DEFAULT_CAMERA.clone();
+      previewCamera.name = "Preview Camera";
```

Comment:
Would it make sense to tag the camera with something more unique? Like setting `previewCamera.userData.spokePreviewCamera=true` or even just `previewCamera.name = "spoke-preview-camera"`?

Reply:
yeah that makes sense -- I was just winging it a bit here since we don't really know how, if at all, we will care about 'preview camera' as a concept. i'll change the name